.github/workflows/bump-channel-version/action.yml

```diff
@@ -44,14 +44,18 @@ runs:
 
             if [[ ${{ inputs.CHANNEL }} == "beta" ]]; then
                 BETA_VERSION=${{ inputs.CLEAN_PACKAGE_VERSION }}-${{ inputs.CHANNEL }}${NEW_CHANNEL_VERSION}
```

Comment:
`BETA_VERSION` can be equal to `ELEMENTOR_CHANNEL_PACKAGE_VERSION`, already defined in line 26.

Reply:
if the channel is Beta than it should be the version number that is being releases
if not it should be an existing beta version